sunpy/lightcurve/sources/het.py

```diff
@@ -0,0 +1,107 @@
+                'Column 13: Uncertainty (sigma) for 17.0-19.3 MeV proton flux',
+                'Column 14: Proton flux, 20.8-23.8 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 15: Uncertainty (sigma) for 20.8-23.8 MeV proton flux',
+                'Column 16: Proton flux, 23.8-26.4 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 17: Uncertainty (sigma) for 23.8-26.4 MeV proton flux',
+                'Column 18: Proton flux, 26.3-29.7 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 19: Uncertainty (sigma) for 26.3-29.7 MeV proton flux',
+                'Column 20: Proton flux, 29.5-33.4 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 21: Uncertainty (sigma) for 29.5-33.4 MeV proton flux' ,
+                'Column 22: Proton flux, 33.4-35.8 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 23: Uncertainty (sigma) for 33.4-35.8 MeV proton flux' ,
+                'Column 24: Proton flux, 35.5-40.5 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 25: Uncertainty (sigma) for 35.5-40.5 MeV proton flux' ,
+                'Column 26: Proton flux, 40.0-60.0 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 27: Uncertainty (sigma) for 40.0-60.0 MeV proton flux' ,
+                'Column 28: Proton flux, 60.0-100.0 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 29: Uncertainty (sigma) for 60.0-100.0 MeV proton flux' ]
+   
+
+    data_modify = []
+    month_dict = {'Jan':1,'Feb':2,'Mar':3,'Apr':4,'May':5,'Jun':6,'Jul':7,'Aug':8,'Sep':9,'Oct':10,'Nov':11,'Dec':12}
+
+    #Storing data columns in recognizable variables (Common to all types of data)
+    start_year_col  = data['col2']
+    start_month_col = data['col3']
+    start_date_col  = data['col4']
+    start_time_col  = data['col5']
+
+    #Adding Time Column based on type of data
+    if type_of_data == '1min':
+        header = ['Verse Number', 'Column 1 : DateTime'] + header 
+
+        for i in range(len(data)): 
+            date = datetime(start_year_col[i], month_dict[ start_month_col[i] ], start_date_col[i], int(("%04d" % (start_time_col[i],))[:2]), int(("%04d" % (start_time_col[i],))[2:]) )
+            data_modify.append(date)
+
+        data.remove_columns(['col{}'.format(i) for i in range(2,6)])
+    else:
+        header = ['Verse Number','Column 1 : TimeRange'] + header
+        
+        #Storing data columns in recognizable variables
+        end_year_col    = data['col6']
+        end_month_col   = data['col7']
+        end_date_col    = data['col8']
+        end_time_col    = data['col9']
+
+        for i in range(len(data)): 
+            date1 = datetime(start_year_col[i], month_dict[ start_month_col[i] ], start_date_col[i], int(("%04d" % (start_time_col[i],))[:2]), int(("%04d" % (start_time_col[i],))[2:]) )
+            date2 = datetime(end_year_col[i], month_dict[ end_month_col[i] ], end_date_col[i], int(("%04d" % (end_time_col[i],))[:2]), int(("%04d" % (end_time_col[i],))[2:]) )
```

Comment:
damn... my previous comment dissapeared.  Anyway, explain why you are converting something into a string and then to an integer...